store/transfer_processor.go

```diff
@@ -196,6 +196,7 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 			s.logger.Error("Failed to get previous balances", "contractAddr", contractAddr.Hex(), "len", len(allAddrs), "err", err)
 			return err
 		}
+		addrDiff[contractAddr] = make(map[gethCommon.Address]*big.Int)
```

Comment:
i think this can be error-prone. if there were previous data in `addrDiff[contractAddr]`, it will be overwritten. i'd like to be more defensive here and only create the map when it doesn't exist.